python/lsst/obs/test/makeTestRawVisitInfo.py

```diff
@@ -19,29 +19,40 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
+__all__ = ["MakeTestRawVisitInfo"]
 
 from lsst.afw.image import VisitInfo, RotType
 from lsst.afw.geom import degrees, SpherePoint
 from lsst.afw.coord import Observatory, Weather
 from lsst.obs.base import MakeRawVisitInfo
 
-__all__ = ["MakeTestRawVisitInfo"]
-
 
 class MakeTestRawVisitInfo(MakeRawVisitInfo):
-    """Make a VisitInfo from the FITS header of a test image
+    """Make a VisitInfo from the FITS header of a test image.
 
+    Notes
+    -----
     Since the test data is extracted from LSST Sim data,
     this is a copy of MakeLsstSimRawVisitInfo
-    (using a copy avoids undesireable dependencies)
+    (using a copy avoids undesireable dependencies).
     """
     observatory = Observatory(-70.749417*degrees, -30.244633*degrees, 2663)  # long, lat, elev
 
     def setArgDict(self, md, argDict):
-        """Set an argument dict for VisitInfo and pop associated metadata
+        """Set an argument dict for VisitInfo and pop associated metadata.
 
-        @param[in,out] md  metadata, as an lsst.daf.base.PropertyList or PropertySet
-        @param[in,out] argdict  a dict of arguments
+        Parameters
+        ----------
+        md : `lsst.daf.base.PropertySet`
+            Image metadata.
+        `dict`
+            A dict of arguments for the `lsst.afw.image.VisitInfo`
+            constructor. Updated by this call.
+
+        Returns
+        -------
+        `lsst.afw.image.VisitInfo`
```

Comment:
Here and below, include a name for the return.

Reply:
OK. That used to be optional but I guess it isn't any longer.